src/client.qc

```diff
@@ -654,7 +654,195 @@ SelectSpawnPoint =
 		if (lastspawn != world)
 			return lastspawn;
 	}
-
+	else if (deathmatch & DM_DM_MASK)
+	{
+	//~ random respawn start
```

Comment:
Roderick attempted this many moons ago and discovered that this would place the player the opposite end of the map usually away from any useful items.  Since then spawns shields were introduced.